### PR TITLE
Cache docs

### PR DESCRIPTION
**Pull request by dbart01**
### What this does

Adds cache documentation. 

cc: @sav007

### CODE REVIEW

**Comment by dbart01** [comment]
@rorytanner Would appreciate it if you could give this a quick look.

**Comment by rorytanner** [comment]
@dbart01 -- edits are included in the last commit

File: README.md [hunk 1]
```diff
@@ -362,6 +363,42 @@ let handler = Graph.RetryHandler<Storefront.QueryRoot>() { (query, error) -> Boo
...
+client.cachePolicy = .cacheFirst
+```
+
+Now, all calls to `queryGraphWith` will yield a task with a `.cacheFirst` cache policy.
+
+If you want to override a client-wide cache policy for an individual request, then specify an alternate cache policy as a parameter of `queryGraphWith`:
+
+```swift
+let task = client.queryGraphWith(query, cachePolicy: .networkFirst(expireIn: 20)) { query, error in
+    // ...
```
Comment (rorytanner) [inline README.md]: is there any content missing here, or can we leave this as `...`?

Comment (dbart01) [inline README.md]: We can leave it as is.

**Comment by dbart01** [comment]
Awesome, thanks @rorytanner !

**Pull request merged**